commit 84fa0df77cb2467521de704e4871daf674626cf5 (refs/changes/49/3149/1)
Author: Mike Zeller <mike.zeller@joyent.com>
Date:   2018-01-02T14:36:50-08:00 (1 year, 9 months ago)
    
    ZAPI-819 Destroying a VM should should not report a failure when there are no NICs to remove

diff --git a/lib/workflows/destroy.js b/lib/workflows/destroy.js
index 3b5d4f4..e8ab082 100644
--- a/lib/workflows/destroy.js
+++ b/lib/workflows/destroy.js
@@ -112,7 +112,7 @@ function removeNics(job, cb) {
      */
 
     if (!macs || !Array.isArray(macs) || macs.length === 0) {
-        cb('No MAC addresses to remove');
+        cb(null, 'No MAC addresses to remove');
         return;
     }
 
